Added Unity Intro

ruiting-chen opened the pull request:
Added page for a basic introduction to Unity
Information on the page includes:
- Download and install Unity Hub and Unity Editor, Licensing
- Create a new Unity project
- Unity user interface

@arielchen07 please review

arielchen07 commented:
Hi! This is great introduction to Unity in general! 
I really like picture for explaining the interface. It's really intuitive and easy for beginners to understand!
Except the suggestion to bold some keywords, i would also recommend to add a section at the end to briefly talk about how to start making the game or just link to some videos. By doing this, the users will know what to do after they finish setting up and get familiared with Unity.

arielchen07 commented:
I just read your comments on my Pull Request and yes, I would also recommend you adding a table of contents at the start of the page like i did!

ruiting-chen commented:
I bolded keywords in my doc and also added a table of contents section.

I think it's a good idea to add a section to talk about how to start making games at the end, to transition readers to start making something useful with Unity. However I think it would be kind of broad to talk about how to start making games in Unity in general. And there will be a lot stuff to write about which will make this doc very long. So I think I will provide links to tutorial video series that demonstrates how to make simple 2D and 3D games in Unity.

arielchen07 commented:
Great! I think referring the readers to some videos is a great idea here! We can just keep this page clean and only focus on some basics.

arielchen07 commented:
I've reread your page after you modified and I left some comments on some other parts. Could you take a look and let me know what you think

ruiting-chen commented:
I added a section for how to get started with making your first Unity game and provided video links for making 2D and 3D games. 
I also changed the title of the page to be more precise. Please review the changes and let me know your thoughts

arielchen07 commented:
I notice that you cannot see my newly-added comments, so I attached the screenshots here.
![Screenshot 2023-11-26 at 7 30 57 PM](https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/105243552/25dc72b3-28b9-45f2-b2b5-c3eee1c4c147)
![Screenshot 2023-11-26 at 7 31 04 PM](https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/105243552/357009e2-0b61-427c-8f15-f280ac8416bd)
![Screenshot 2023-11-26 at 7 31 11 PM](https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/105243552/5bbfda2e-5e65-438f-8315-11f3e2b66d04)

I reviewed your new changes, it seems that you fixed the first and second comment that I mentioned, which is great! I also like your separate section for how to get started with making your first Unity game. It makes the whole page more structural.

By saying these, could also take a look at my comments in the third screenshot?

ruiting-chen commented:
ohh I see. Yeah I agree with you that the links in reference are a bit messy, I will fix that

ruiting-chen commented:
I updated the reference links. Please have a look and let me know how they look now

arielchen07 reviewed (commented):
Great! It looks much organized now!

ruiting-chen commented on Topics/Tech_Stacks/Unity_Intro.md:
```diff
@@ -0,0 +1,78 @@
+
+#### License Activation:
+An activated license is needed to use Unity. When you sign in to the Unity Hub for the first time, you will be asked to add an active license. 
+At the top of Unity Hub, click "Manage licenses". If you do not see this, you can manage licenses by clicking on the preferences (or settings) icon on the top of the left tab bar beside your account icon > then select the license tab in the window that pops up. 
```
updated it

arielchen07 commented:
One last thing before merging this PR, do we need to add the name of this page to the readme file on the wiki?

ruiting-chen commented on Topics/Tech_Stacks/Unity_Intro.md:
```diff
@@ -0,0 +1,98 @@
+
+https://docs.unity3d.com/Manual/VisualStudioIntegration.html#:~:text=Unity%20automatically%20uses%20Visual%20Studio,into%20an%20existing%20Unity%20installation. (Unity Manual)
+
+https://getalow.com/unity-engine/introduction-to-unity-editor-and-unity-interface/16 (Introduction to Unity Editor and Unity Interface)
```
I updated the format

arielchen07 commented:
I think it's in the FAQ
> What are some of the best practices to keep in mind for submission?
i. Give relevant names to your commit messages
ii. Summarize your contributions in the comment section of your PR.
iii. Document your approach wherever relevant. This ensures the
ease of use and understanding of your contribution for other
students; for example, if you request changes to someone else’s
PR- clearly explain your thought process. Update the readme files
on the wiki wherever appropriate.

ruiting-chen commented:
I added the page name in readme and Tech_Stacks.md

ruiting-chen commented:
ok! i will merge the pull request now
